blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,347 @@
+        )
+        : null }
+      {(promoElements.showDescription || promoElements.showDate || promoElements.showByline)
+        ? (
+          <div className="results-list--description-author-container mobile-order-3">
+            {(promoElements.showDescription && descriptionText)
+              ? (
+                <a href={url} title={headlineText}>
+                  <SecondaryFont as="p" className="description-text">
+                    {descriptionText}
+                  </SecondaryFont>
+                </a>
+              )
+              : null}
+            {(promoElements.showDate || promoElements.showByline)
+              ? (
+                <div className="results-list--author-date">
+                  {(promoElements.showByline)
+                    ? (
+                      <Byline
+                        content={element}
+                        list
+                        separator={promoElements.showDate}
+                        font="Primary"
+                      />
+                    )
+                    : null }
+                  {(promoElements.showDate)
+                    ? <ArticleDate classNames="story-date" date={displayDate} />
+                    : null }
+                </div>
+              )
+              : null}
+          </div>
+        )
+        : null }
+    </div>
+  );
+}));
+
+const Results = ({
+  customFields,
+  fusionContext,
+  fusionProperties,
+  imageProperties,
+}) => {
+  const {
+    lazyLoad,
+    listContentConfig: {
+      contentService,
+      contentConfigValues,
+    },
+  } = customFields;
+
+  const {
+    arcSite,
+    contextPath,
+    deployment,
+    isAdmin,
+  } = fusionContext;
+
+  const {
+    fallbackImage,
+    locale,
+  } = fusionProperties;
+
+  const [configuredOffset] = useState(
+    parseInt(contentConfigValues.offset, 10)
+    || parseInt(contentConfigValues.feedOffset, 10)
+    || parseInt(contentConfigValues.from, 10)
+    || 0,
+  );
+  const [configuredSize] = useState(
+    parseInt(contentConfigValues.size, 10)
+    || parseInt(contentConfigValues.feedSize, 10)
+    || 10,
+  );
+
+  const [queryOffset, setQueryOffset] = useState(configuredOffset);
+
+  const [targetFallbackImage] = useState(
+    !(fallbackImage.includes('http'))
+      ? deployment(`${contextPath}/${fallbackImage}`)
+      : fallbackImage,
+  );
+
+  const placeholderResizedImageOptions = useContent({
+    source: !targetFallbackImage.includes('/resources/') ? 'resize-image-api' : null,
+    query: { raw_image_url: targetFallbackImage, respect_aspect_ratio: true },
+  });
+
+  const [isServerSideLazy] = useState(lazyLoad && isServerSide() && !isAdmin);
+
+  const serviceQueryPage = useCallback((requestedOffset = 0) => {
+    /*
+      This sets up a window view of the data starting from the initial index
+      to *twice* the size.  When clicking on the more button, we will render
+      the next size worth of items and load the next size from the server.
```

Comment:
this is different behavior than what we had initially, right? Assuming so, this decision to load more data than is necessary will have the benefit of faster functionality later. But there's the tradeoff that this will slow down the first render. Was this part of the intention of the ticket?

Reply:
Heh, this was the initial concern for me on the ticket.  Previously, the feeds that did not have a `next` property would "preload" the next set of results to determine if the see more button should be shown, then they were thrown away.  So I thought it would be a better use experience and less wasteful to preload the next set of results so when the user hit the button there was no delay in showing them the results and we could load the next set in the background.

Reply:
done